app/models/concerns/authenticateable.rb

```diff
@@ -13,17 +13,19 @@ def password_is_valid?(password)
     salt = salted_password.split("$")[2]
 
     salted_password == salt_password(salt, password)
-  rescue StandardError, Cheetah::ExecutionFailed
+  rescue StandardError, Cheetah::ExecutionFailed => e
     return false
   end
 
   private
 
   def salt_password(salt, password)
-    Cheetah.run(
+    stdout, stderr = Cheetah.run(
```

Comment:
should we not check `stderr` ?

Reply:
stdout & stderr is only populated if there is no exception (e.g. exit code 0). It's unlikely there's anything useful in stderr.

Reply:
OK, so `openssl passwd` does not populate `stderr` cool